<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rec2Track</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>29</y>
          <width>233</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="1">
          <widget class="QSpinBox" name="repeats"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Repeats</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Visual Stimulation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>Tom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rachel</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dominic</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>51</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Folder</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="FolderEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
